app/questionnaire/placeholder_transforms.py

```diff
@@ -147,6 +143,42 @@ def format_ordinal(number_to_format, determiner=None):
 
         return f"{number_to_format}{suffix}"
 
+    @staticmethod
+    def get_ordinal(language_code, number_to_format):
```

Comment:
Shouldn't be static, as it means you don't need to pass language code. Also, method name should be something like `get_suffix_for_ordinal`.

Reply:
I renamed it to use `indicator` as `suffix` is not the correct term